From 75e92438bcdb78a90912366b07bf503646806686 Mon Sep 17 00:00:00 2001
From: "Arnout Vandecappelle (Essensium/Mind)" <arnout@mind.be>
Date: Sun, 21 Feb 2016 17:02:06 +0100
Subject: [PATCH] configure.ac: check for localtime_r

HAVE_LOCALTIME_R is used in owftp.c, so it should be checked for.

Without this, static build fails because localtime_r is defined twice.

Upstream-Status: Submitted
https://sourceforge.net/p/owfs/mailman/message/34873667/

Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>
[Bernd: rebased against version 3.2p1]
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index c22dde9..12fdd18 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1646,7 +1646,7 @@
 AC_FUNC_STRFTIME
 AC_FUNC_STRTOD
 AC_TYPE_SIGNAL
-AC_CHECK_FUNCS([accept daemon getaddrinfo freeaddrinfo gethostbyname2_r gethostbyaddr_r gethostbyname_r getservbyname_r getopt getopt_long gmtime_r gettimeofday localtime_r inet_ntop inet_pton memchr memset select socket strcasecmp strchr strdup strncasecmp strtol strtoul twalk tsearch tfind tdelete tdestroy vasprintf strsep vsprintf vsnprintf writev getline])
+AC_CHECK_FUNCS([accept daemon getaddrinfo freeaddrinfo gethostbyname2_r gethostbyaddr_r gethostbyname_r getservbyname_r getopt getopt_long gmtime_r gettimeofday localtime_r inet_ntop inet_pton memchr memset select socket strcasecmp strchr strdup strncasecmp strtol strtoul twalk tsearch tfind tdelete tdestroy vasprintf strsep vsprintf vsnprintf writev getline localtime_r])
 
 save_LIBS="$LIBS"
 LIBS=""
-- 
2.7.0

